app/src/main/kotlin/edu/stanford/bdh/engagehf/MainActivity.kt

```diff
@@ -48,7 +49,7 @@ import javax.inject.Inject
 import kotlin.reflect.typeOf
 
 @AndroidEntryPoint
-class MainActivity : ComponentActivity() {
+class MainActivity : FragmentActivity() {
```

Comment:
Can you please also update the `ComposeContentActivity` to extend `FragmentActivity` as well? We use it for testing and it would be great to have the same behaviour